Resources/Prototypes/Entities/Structures/Specific/anomalies.yml

```diff
@@ -231,3 +231,140 @@
     releasedGas: 8 # Frezon. Please replace if there is a better way to specify this
     releaseOnMaxSeverity: true
     spawnRadius: 0
+
+- type: entity
+  id: AnomalyLiquid
+  parent: BaseAnomaly
+  suffix: Liquid
+  components:
+  - type: Sprite
+    sprite: Structures/Specific/Anomalies/liquid_anom.rsi
+    layers:
+    - state: anom
+      map: ["enum.AnomalyVisualLayers.Base"]
+    - state: pulse
+      map: ["enum.AnomalyVisualLayers.Animated"]
+      visible: false
+  - type: RandomSprite
+    selected: # Initialized layer values. Edited through the ReagentProducerAnomalySystem
+      enum.AnomalyVisualLayers.Base:
+        anom: "#ffffff"
+      enum.AnomalyVisualLayers.Animated:
+        pulse: "#ffffff"
+  - type: PointLight
+    radius: 4.0
+    energy: 3.5
+    color: "#bbbbbb"
+  - type: BadFood
+  - type: Anomaly
+    anomalyContactDamage:
+      types:
+        Slash: 1
+  #- type: ExplosionAnomaly
+  #  supercriticalExplosion: Default
+  #  explosionTotalIntensity: 50
+  #  explosionDropoff: 2
+  #  explosionMaxTileIntensity: 10
+  - type: EntitySpawnAnomaly
+    superCriticalSpawn: ReagentSlimeSpawner
+    spawns:
+      - PuddleSparkle
+  - type: SolutionContainerManager
+    solutions:
+      anomaly:
+        maxVol: 1500
+  - type: PuddleCreateAnomaly
+    solution: anomaly
+  - type: InjectionAnomaly
+    solution: anomaly
+  - type: ReagentProducerAnomaly
+    solution: anomaly
+    needRecolor: true
+    blacklistChemicals: 
+    # With a limit of 15u injections are too strong
+    - Romerol           
+    - Bungotoxin
+    - Lead
+    - VentCrud
+    - Lexorin
+    # OP Medical        
+    - AmbuzolPlus
+    - Siderlac
+    - Omnizine
+    # Fun Category
+    - Fiber             
+    - GroundBee
+    - ZombieBlood   # AmbusolPlus again
+    - Ichor
+    - Stimulants
+    # Gases - Group Blacklist (and move all gases into groud)
+    - Oxygen
+    - Plasma
+    - Tritium
+    - CarbonDioxide
+    - Nitrogen
+    - Miasma
+    - NitrousOxide
+    - Frezon
+    # base
+    - BasePyrotechnic
```

Comment:
basepyrotechnic etc should not be required to be blacklisted--should be possible to exclude these by just checking if the reagentprototype has `Abstract` true

Reply:
I'm just removing the "Other" category, leaving only a whitelist of fun, usefual and dangerous. And no abstract reagents will be writing there.